packages/editor-ui/src/utils/objectUtils.ts

```diff
@@ -0,0 +1,20 @@
+type ObjectOrArray = Record<string, unknown> | unknown[];
+
+export function isDateObject(maybeDate: unknown): maybeDate is Date {
+	return maybeDate instanceof Date;
+}
+
+export function isObjectOrArray(maybeObject: unknown): maybeObject is ObjectOrArray {
+	return typeof maybeObject === 'object' && maybeObject !== null && !isDateObject(maybeObject);
+}
+
+export function isObject(maybeObject: unknown): maybeObject is ObjectOrArray {
```

Comment:
This should be `maybeObject  is Record<string, unknown>`.

Reply:
Eagle eye :)
Fixed